salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/stepdefs/LoginSteps.java

```diff
@@ -41,15 +44,15 @@ public LoginSteps(final Context sharedContext) {
     public void logInToSalesforce(final String userAlias) throws IOException, PropertiesReadingException {
 
         //To run only one scenario uncomment the lines below
-        //TestExecutionProperties.setRootPath("../salesforce-core/");
-        //TestPropertiesSetter.setDataProviderThreadCountProp();
-        //TestPropertiesSetter.setTestBrowser();
+        TestExecutionProperties.setRootPath("../salesforce-core/");
+        TestPropertiesSetter.setDataProviderThreadCountProp();
+        TestPropertiesSetter.setTestBrowser();
 
         //Updating User Entity to get credentials
         user = context.getUserByAlias(userAlias);
 
         //Set User Authentication to use Salesforce API in next steps
-        RequestManager.setRequestSpec(ApiAuthenticator.getLoggedReqSpec(user));
+        context.setRequestSpec(ApiAuthenticator.getLoggedReqSpec(user));
```

Comment:
Why did you carry this method to the context?? Is that its responsability?? (Remember that this set "LoggedReqSpec")